spec/models/core/location_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Core::Location, type: :model do
+
+  context 'associations' do
+    it {is_expected.to have_many(:events)}
+  end
+
+  context '.save' do
+    it 'succeeds' do
+      test_location = build(:core_location)
+      expect(test_location.save).to be_truthy
```

Comment:
change to `expect(text_location.save).to be(true)`. This function returns true if successful. Of course run the test after making the change and committing.